src/main/java/frc5124/robot2020/subsystems/DriveTrain.java

```diff
@@ -101,7 +108,7 @@ public void periodic() {
     // Control methods
 
     public void tankDrive(double left, double right) {
-        differentialDrive.tankDrive(left,right);   
+        differentialDrive.tankDrive(left,-right);   
```

Comment:
Differential drive should take care of the negation: are you sure this is right? We should be able to get this to work using the motor directions or something.